plugins/node/opentelemetry-instrumentation-fastify/test/instrumentation.test.ts

```diff
@@ -137,7 +137,7 @@ describe('fastify', () => {
       const span = spans[3];
       assert.deepStrictEqual(span.attributes, {
         'fastify.type': 'request_handler',
-        'plugin.name': 'fastify-express',
+        'plugin.name': 'fastify -> @fastify/express',
```

Comment:
Fastly name spaces plugins consistently now, showing the hierarchy, as it could be ambiguous otherwise.